sdk/core/abort-controller/review/abort-controller.api.md

```diff
@@ -21,10 +21,11 @@ export class AbortError extends Error {
 // @public
 export class AbortSignal implements AbortSignalLike {
     constructor();
-    readonly aborted: boolean;
+    get aborted(): boolean;
     addEventListener(_type: "abort", listener: (this: AbortSignalLike, ev: any) => any): void;
-    static readonly none: AbortSignal;
-    onabort?: (ev?: Event) => any;
+    dispatchEvent(_event: Event): boolean;
```

Comment:
The other four changes seem reasonable if weird, but here `dispatchEvent` has just appeared out of thin air. Did we modify this signature somewhere to export it?